metrics/grafana/overview.json

```diff
@@ -235,6 +235,13 @@
               "intervalFactor": 2,
               "legendFormat": "Kafka_exporter",
               "refId": "L"
+            },
+            {
+              "expr": "\ncount(probe_success{group=\"tiflash\"} == 1)",
```

Comment:
Why not move it below the `tikv`?

Reply:
so your suggestion is to move `tiflash` below `tikv`?

Reply:
Yes.

Reply:
fixed